### PR TITLE
CI: fix flatpak action

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -6,13 +6,14 @@ name: CI
 jobs:
   flatpak-builder:
     name: "Flatpak Builder"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     container:
-      image: bilelmoussaoui/flatpak-github-actions:gnome-3-38
+      image: bilelmoussaoui/flatpak-github-actions:gnome-3.38
```
Comment (bleakgrey) [inline .github/workflows/build.yml:11]: :')

Comment (bilelmoussaoui) [reply .github/workflows/build.yml:11]: ?

Comment (bleakgrey) [reply .github/workflows/build.yml:11]: I spent an hour wondering why this doesn't work and all I had to do is fix one symbol. I find it extremely hilarious 😄

Comment (bilelmoussaoui) [reply .github/workflows/build.yml:11]: Ah it was a mistake on our side :P

Comment (bilelmoussaoui) [reply .github/workflows/build.yml:11]: (the docs were wrong)

**Pull request merged**